src/index.ts

```diff
@@ -244,39 +246,42 @@ class TestcafeTestrailReporter {
               console.error(`[TestRail] All TestRail mappings should be valid. Following test case id does not exist in TestRail: ${id}.`);
             }
           });
-
-          const { value: runs } = await throwOnApiError(testrailAPI.getRuns(projectId, { is_completed: 0 }));
-          const existingRun = runs?.find((run) => run.refs === refs);
-
+          let existingRun: Run | undefined;
           let run: Run;
-          if (existingRun) {
-            run = existingRun;
-            const { value: tests } = await throwOnApiError(testrailAPI.getTests(existingRun.id));
-            const currentCaseIds = tests?.map((test) => test.case_id) || [];
-            const additionalDescription = "\n" + runDescription;
-
-            await throwOnApiError(
-              testrailAPI.updateRun(existingRun.id, {
-                description: existingRun.description.replace(additionalDescription, "") + additionalDescription,
-                case_ids: [...currentCaseIds, ...caseIdList],
-              })
-            );
-
-            console.log(`[TestRail] Test run updated successfully: ${runName}`);
+          if(runId) {
+            const { value: returnedRun } = await throwOnApiError(testrailAPI.getRun(runId));
+            run = returnedRun

```

Comment:
I guess it would be better to treat this as existing run, so test cases could be updated automatically.

Reply:
Yes, but I had a problem because I need to no update the current run, but was not possible to me make true as the default value, i'll change it and will create a new variable for disable update